src/shared/src/types/project-types.ts

```diff
@@ -35,6 +35,12 @@ export enum WbsElementStatus {
   Complete = 'COMPLETE'
 }
 
+export enum WPFormType {
+  Create = 'CREATE',
+  Edit = 'EDIT',
+  CrCreate = 'CrCREATE'
```

Comment:
enum names should be capitalized (CREATE, EDIT, etc.), also rename to CREATEWITHCR, it's a bit more specific